scripts/one-docker-to-rule-them-all.sh

```diff
@@ -6,22 +6,37 @@ cleanup() {
   echo "stopping geth docker image"
   docker kill geth
   yarn workspace @augurproject/gnosis-relay-api kill-relay
+  docker volume prune --force
+  docker system prune --force
```

Comment:
I worry about taking such destructive action inside a script that isn't obviously this destructive.

Reply:
ok, I'll remove